.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,81 @@
+---
+name: "\U0001F41B Bug Report"
+about: "Please report issues related to the TypeScript and HTML API of Vaadin components here."
+title: ''
+labels: 'i: needs triage'
+assignees: ''
+
+---
+
+<!--
+  Please fill out each section below, otherwise, your issue risks taking an infinite amount of time to triage. This info allows the maintainers team to diagnose and fix your issue as quickly as possible.
+
+  Before opening a new issue, please search existing issues: https://github.com/vaadin/web-components/issues.
+-->
+
+### Description
+
+Describe the issue that you're seeing.
+
+<!-- Example: The `vaadin-grid` element scrolls when applying filter. -->
+
+### Expected outcome
+
+What should happen?
+
+<!-- Example: No scrolling should happen. -->
+
+### Actual outcome
+
+What happened.
+
+<!-- Example: Grid content occasionally gets scrolled. -->
+
+### Live Demo
+
+A screenshot or video with issue reproduced.
+
+### Minimal reproducible example
+
+**Please** provide a code snippet with issue reproduction, this makes your issue _much_ easier to diagnose.
+
+<!--
+```
+<vaadin-grid</vaadin-grid>
+```
+```
+const grid = document.querySelector('vaadin-grid');
+grid.items = items;
+```
+-->
+
+### Steps to reproduce
+
+Clear steps describing how to reproduce the issue.
```

Comment:
I'm thinking that this text could also refer to the reproduction that _should_ be provided above. 

The example below would then just be:
```
3. Open the page in a web browser.
4. Start typing in the filter input
```

Reply:
I would like to preserve the details for the reproduction if possible (with initial steps, like adding `grid` and `grid-filter` to the page). So the user will provide as many details as possible when creating `Steps to reproduce`.

Reply:
OK sure.